Libraries/HyperV.psm1

```diff
@@ -37,7 +37,12 @@ function Create-AllHyperVGroupDeployments($SetupTypeData, $GlobalConfig, $TestLo
         $HyperVHostArray = @()
         if ($deployOnDifferentHosts -eq "yes") {
             foreach ($HypervHost in $GlobalConfig.Global.HyperV.Hosts.ChildNodes) {
-                $HyperVHostArray += $HyperVHost.ServerName
+                if (!($HyperVHostArray -contains $HyperVHost.ServerName)) {
+                    $HyperVHostArray += $HyperVHost.ServerName
+                }
```

Comment:
} else {

Reply:
formatting by official powershell extention, it will be in a new line.

Reply:
LISAv2 has its own format. Can you follow ours?

Reply:
I think it is important to keep the rule in the open source project. We all follow, so do you.